code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -198,7 +198,6 @@
 		/obj/item/clothing/gloves/latex,
 		/obj/item/reagent_containers/hypospray,
 		/obj/item/clothing/glasses/hud/health,
-		/obj/item/crowbar,
```

Comment:
I'm not seeing other code I could use as an example at the moment. Would it just be `can_hold_blacklist = /obj/item/crowbar/rescue_axe` ? `pickup_blacklist` implies in its comment it's for stopping items from being picked up by the container if you click on them, not a way to prevent them from being stored.